korlibs-serialization/src/korlibs/io/serialization/xml/Xml.kt

```diff
@@ -397,11 +397,19 @@ data class Xml(
                 val openclose = r.tryExpect('/')
                 val processingInstructionEnd = r.tryExpect('?')
                 r.skipExpect('>')
+
+                // Handle namespace processing based on the processNamespaces flag
+                val elementName = if (!processNamespaces && name.contains(':')) {
+                    name.split(':').last()
```

Comment:
To make it faster and reduce allocations, can you change it to:

```suggestion
                    name.substringAfter(':')
```